README.md

```diff
@@ -1,4 +1,6 @@
-# The Distribution Mapping and Analysis Portal (DisMAP) 
+# [The Distribution Mapping and Analysis Portal (DisMAP)](https://github.com/nmfs-fish-tools/DisMAP) <img src="https://www.google.com/url?sa=i&url=https%3A%2F%2Fapps-st.fisheries.noaa.gov%2Fdismap%2F&psig=AOvVaw3zIJ4BQICcgR2yTEvttKwd&ust=1713558777248000&source=images&cd=vfe&opi=89978449&ved=0CBIQjRxqFwoTCPDm_oPOzIUDFQAAAAAdAAAAABAE" alt="Logo." align="right" width="139" height="139"/>
```

Comment:
this figure isn't loading. I am going to delete the code for the logo insertion.

Reply:
Oh, good catch! thanks for taking care of that.